packages/ui/src/components/Command/Command.utils.tsx

```diff
@@ -358,14 +358,14 @@ export function useHistoryKeys({ enable, messages, setPrompt }: UseHistoryKeysOp
         case 'ArrowUp':
           setMessageSelectionIndex((index) => {
             const newIndex = Math.max(index - 1, 0)
-            setPrompt(messages[newIndex] ?? '')
+            if (messages[newIndex]) setPrompt(messages[newIndex])
```

Comment:
For additional clarity & prevent future errors:
```suggestion
            const newMessage = messages[newIndex]
            if (newMessage) {
              setPrompt(newMessage)
            }
```